doc/rst/tools/chpl-language-server/chpl-language-server.rst

```diff
@@ -69,7 +69,9 @@ With Emacs 29.1, support has been added for language server protocols via `Eglot
 <https://www.gnu.org/software/emacs/manual/html_mono/eglot.html>`_
 
 To utilize the Chapel language server with Eglot, add the following to your
-``.emacs`` file:
+``.emacs`` file (note that this assumes you have already followed the
+instructions in ``$CHPL_HOME/highlight/emacs/README.rst`` to install Chapel
+syntax highlighting in emacs):
```

Comment:
nittest of all nits, but emacs is lowercase here and uppercase in the other one :)

Reply:
Dangnabit

Reply:
I didn't save the file